operator/pkg/model/ingestion/gamma.go

```diff
@@ -87,16 +87,15 @@ func GammaHTTPRoutes(input GammaInput) []model.HTTPListener {
 
 			parentSvc, err := getMatchingService(parentName.Name, parentName.Namespace, hr.GetNamespace(), input.Services)
 			if err != nil {
-				fmt.Printf("Can't find parent Service %s/%s in input\n", parentName.Namespace, parentName.Name)
+				log.Warnf("Can't find parent Service %s/%s in input", parentName.Namespace, parentName.Name)
 				continue
 			}
 
 			if parentSvc.GetName() == "" {
 				// skip processing this parent because it's not in the input.
 				// This situation should not arise - there should be multiple
 				// layers of protection.
-				// TODO: figure out how to handle this.
-				fmt.Printf("Can't find any parent Service in input\n")
```

Comment:
If any of these Warnings show up in the logs, how can users know what to do in order for them to fix the issue?